README.md

```diff
@@ -16,7 +16,8 @@ Feel free to add issues in github or send pull requests.
 ### Rules for commits
 
 * Every commit is checked by [Travis CI](https://travis-ci.com/) as soon as
-  you create a pull request but you *should* run `test_logwarn` locally,
+  you create a pull request.
+  You can locally run `./test_logwarn` or `prove test_logwarn` to run the tests yourself.
```

Comment:
please wrap lines consistently at 80 characters.

But also I consider the phrasing not so good. It says "you can run … to run". How about just


```suggestion
 * Every commit is checked by
 [CI](https://github.com/os-autoinst/openqa-logwarn/actions) as soon as you
 create a pull request but you *should* run tests locally.
```

As the usage section already explains how to call tests so please don't repeat the usage in the rules for commit

Reply:
I honestly think the previous phrasing was fine and maybe should just be extended with the additional `prove test_logwarn`

Reply:
For perspective, I actually read this as a checklist, especially as a new contributor. And my brain parsed "please test manually if you know how". **Usage** is not where I'd look at all.

Reply:
maybe we could link here to the **Usage** section to avoid duplication but also have it included in the "checklist"?

Reply:
Done as suggested and expanded the existing section a bit